### PR TITLE
[docs] Clean the pickers migration guide

### PR DESCRIPTION
**Pull request by flaviendelangle**

### CODE REVIEW

File: docs/data/migration/migration-pickers-v6/migration-pickers-v6.md [hunk 1]
```diff
@@ -636,7 +647,8 @@ The `isNull` method have been removed, you can replace it with a very basic chec
 
 #### Remove the `mergeDateAndTime` method
 
-The `mergeDateAndTime` method have been removed, you can use the `setHours`, `setMinutes`, and `setSeconds` methods to recreate its behavior:
+The `mergeDateAndTime` method have been removed.
+You can use the `setHours`, `setMinutes`, and `setSeconds` methods to recreate its behavior:
```
Comment (flaviendelangle) [inline docs/data/migration/migration-pickers-v6/migration-pickers-v6.md:651]: Me IDE was suggesting me to do it on some longer sentences to split it in two
I applied it everywhere, WDYT?

File: docs/data/migration/migration-pickers-v6/migration-pickers-v6.md [hunk 2]
```diff
@@ -657,7 +669,8 @@ The `mergeDateAndTime` method have been removed, you can use the `setHours`, `se
 
 #### Remove the `parseISO` method
 
-The `parseISO` method have been removed, you can directly use your date library:
+The `parseISO` method have been removed.
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v6/migration-pickers-v6.md:672]:
```suggestion
The `parseISO` method has been removed.
```

File: docs/data/migration/migration-pickers-v6/migration-pickers-v6.md [hunk 3]
```diff
@@ -679,13 +692,24 @@ The `parseISO` method have been removed, you can directly use your date library:
 
 #### Remove the `toISO` method
 
-The `toISO` method have been removed, you can directly use your date library:
+The `toISO` method have been removed.
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v6/migration-pickers-v6.md:695]:
```suggestion
The `toISO` method has been removed.
```

File: docs/data/migration/migration-pickers-v6/migration-pickers-v6.md [hunk 4]
```diff
@@ -474,7 +476,8 @@ You can use the `date` method instead:
 
 #### Remove the `getDiff` method
 
-The `getDiff` method have been removed, you can directly use your date library:
+The `getDiff` method have been removed.
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v6/migration-pickers-v6.md:479]:
```suggestion
The `getDiff` method has been removed.
```

File: docs/data/migration/migration-pickers-v6/migration-pickers-v6.md [hunk 5]
```diff
@@ -531,14 +534,16 @@ The `getDiff` method have been removed, you can directly use your date library:
 
 #### Remove the `getFormatHelperText` method
 
-The `getFormatHelperText` method have been removed, you can use the `expandFormat` instead:
+The `getFormatHelperText` method have been removed.
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v6/migration-pickers-v6.md:537]:
```suggestion
The `getFormatHelperText` method has been removed.
```

File: docs/data/migration/migration-pickers-v6/migration-pickers-v6.md [hunk 6]
```diff
@@ -560,7 +565,8 @@ And if you need the exact same output, you can apply the following transformatio
 
 #### Remove the `getMeridiemText` method
 
-The `getMeridiemText` method have been removed, you can use the `setHours`, `date` and `format` methods to recreate its behavior:
+The `getMeridiemText` method have been removed.
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v6/migration-pickers-v6.md:568]:
```suggestion
The `getMeridiemText` method has been removed.
```

File: docs/data/migration/migration-pickers-v6/migration-pickers-v6.md [hunk 7]
```diff
@@ -574,7 +580,8 @@ The `getMeridiemText` method have been removed, you can use the `setHours`, `dat
 
 #### Remove the `getMonthArray` method
 
-The `getMonthArray` method have been removed, you can use the `startOfYear` and `addMonths` methods to recreate its behavior:
+The `getMonthArray` method have been removed.
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v6/migration-pickers-v6.md:583]:
```suggestion
The `getMonthArray` method has been removed.
```

File: docs/data/migration/migration-pickers-v6/migration-pickers-v6.md [hunk 8]
```diff
@@ -595,7 +602,8 @@ The `getMonthArray` method have been removed, you can use the `startOfYear` and
 
 #### Remove the `getNextMonth` method
 
-The `getNextMonth` method have been removed, you can use the `addMonths` method instead:
+The `getNextMonth` method have been removed.
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v6/migration-pickers-v6.md:605]:
```suggestion
The `getNextMonth` method has been removed.
```

File: docs/data/migration/migration-pickers-v6/migration-pickers-v6.md [hunk 9]
```diff
@@ -604,7 +612,8 @@ The `getNextMonth` method have been removed, you can use the `addMonths` method
 
 #### Remove the `getPreviousMonth` method
 
-The `getPreviousMonth` method have been removed, you can use the `addMonths` method instead:
+The `getPreviousMonth` method have been removed.
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v6/migration-pickers-v6.md:615]:
```suggestion
The `getPreviousMonth` method has been removed.
```

File: docs/data/migration/migration-pickers-v6/migration-pickers-v6.md [hunk 10]
```diff
@@ -613,7 +622,8 @@ The `getPreviousMonth` method have been removed, you can use the `addMonths` met
 
 #### Remove the `getWeekdays` method
 
-The `getWeekdays` method have been removed, you can use the `startOfWeek` and `addDays` methods instead:
+The `getWeekdays` method have been removed.
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v6/migration-pickers-v6.md:625]:
```suggestion
The `getWeekdays` method has been removed.
```

File: docs/data/migration/migration-pickers-v6/migration-pickers-v6.md [hunk 11]
```diff
@@ -627,7 +637,8 @@ The `getWeekdays` method have been removed, you can use the `startOfWeek` and `a
 
 #### Remove the `isNull` method
 
-The `isNull` method have been removed, you can replace it with a very basic check:
+The `isNull` method have been removed.
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v6/migration-pickers-v6.md:640]:
```suggestion
The `isNull` method has been removed.
```

File: docs/data/migration/migration-pickers-v6/migration-pickers-v6.md [hunk 12]
```diff
@@ -636,7 +647,8 @@ The `isNull` method have been removed, you can replace it with a very basic chec
 
 #### Remove the `mergeDateAndTime` method
 
-The `mergeDateAndTime` method have been removed, you can use the `setHours`, `setMinutes`, and `setSeconds` methods to recreate its behavior:
+The `mergeDateAndTime` method have been removed.
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v6/migration-pickers-v6.md:650]:
```suggestion
The `mergeDateAndTime` method has been removed.
```

File: docs/data/migration/migration-pickers-v6/migration-pickers-v6.md (hunk 1 above)
Comment (LukasTy) [reply docs/data/migration/migration-pickers-v6/migration-pickers-v6.md:651]: Makes a lot of sense! 👏 👍

**Review by LukasTy — APPROVED** [review]

File: docs/data/migration/migration-pickers-v6/migration-pickers-v6.md [hunk 13]
```diff
@@ -679,13 +692,24 @@ The `parseISO` method have been removed, you can directly use your date library:
 
 #### Remove the `toISO` method
 
-The `toISO` method have been removed, you can directly use your date library:
+The `toISO` method has been removed.
+You can directly use your date library:
 
 ```diff
+// For Day.js
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v6/migration-pickers-v6.md:699]:
```suggestion
 // For Day.js
```

File: docs/data/migration/migration-pickers-v6/migration-pickers-v6.md [hunk 14]
```diff
@@ -679,13 +692,24 @@ The `parseISO` method have been removed, you can directly use your date library:
...
 -const isoString = adapter.toISO(value);
 +const isoString = value.toISOString();
+
+// For Luxon
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v6/migration-pickers-v6.md:703]:
```suggestion
 // For Luxon
```

File: docs/data/migration/migration-pickers-v6/migration-pickers-v6.md [hunk 15]
```diff
@@ -679,13 +692,24 @@ The `parseISO` method have been removed, you can directly use your date library:
...
+-const isoString = adapter.toISO(value);
 +const isoString = value.toUTC().toISO({ format: 'extended' });
+
+// For DateFns
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v6/migration-pickers-v6.md:707]:
```suggestion
 // For DateFns
```

File: docs/data/migration/migration-pickers-v6/migration-pickers-v6.md [hunk 16]
```diff
@@ -679,13 +692,24 @@ The `parseISO` method have been removed, you can directly use your date library:
...
+-const isoString = adapter.toISO(value);
 +const isoString = dateFns.formatISO(value, { format: 'extended' });
+
+// For Moment
```
Comment (LukasTy) [inline docs/data/migration/migration-pickers-v6/migration-pickers-v6.md:711]:
```suggestion
 // For Moment
```

**Review by LukasTy — APPROVED** [review]
Applying these should make CI happy. 🤞 🙈

**Pull request merged**